Resources/Private/Templates/CompactView/Index.html

```diff
@@ -1,19 +1,17 @@
-<link rel="stylesheet" type="text/css"
-      href="https://d8ejoa1fys2rk.cloudfront.net/modules/compactview/includes/css/compactview_global-1.0.1.css">
-
 <div id="bynder-compactview"
-     data-language="{language}"
-     data-button="Pick a file from Bynder"
```

Comment:
Why did you drop the button text?

Reply:
Can you give me an example, where this is shown? The inline button uses this label. I cannot see where this is used in this compact view :-)